drivers/windows/windows.go

```diff
@@ -457,20 +471,21 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		VirtualNetwork: n.config.HnsID,
 	}
 
-	ec, err := parseEndpointOptions(epOptions)
+	epOption, err := parseEndpointOptions(epOptions)
+	epConnectivity, err := parseEndpointConnectivity(epOptions)
```

Comment:
Is it really not possible for windows to implement the `ProgramExternalConnectivity()` function ?
From this diffs, it is still relaying on the old model of parsing the external connectivity options during endpoint creation.